cod/deploy.yaml

```
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cod
+  namespace: w3cie
+  labels:
+    k8s-app: cod
+  annotations:
+    fluxcd.io/automated: 'true'
+spec:
+  replicas: 2
+  revisionHistoryLimit: 5
+  selector:
+    matchLabels:
+      k8s-app: cod
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+    type: RollingUpdate
+  template:
+    metadata:
+      name: cod
+      labels:
+        k8s-app: cod
+    spec:
+      containers:
+      - name: cod
+        image: quay.io/wisvch/cod:49 #49?
```

This is not right. There is no build tagged in https://quay.io/repository/wisvch/cod?tab=tags yet.

---

Tag is 8

---

Latest build tagged is now 8: https://quay.io/repository/wisvch/cod?tab=tags

```suggestion
        image: quay.io/wisvch/cod:8
```